src/app/parcels/parcelsTable/styles.ts

```diff
@@ -11,38 +11,42 @@ export const PreTableControls = styled.div`
 
 export const parcelTableColumnStyleOptions = {
     iconsColumn: {
-        width: "4rem",
+        maxWidth: "4rem",
+        minWidth: "0.7rem",
+        grow: 0.5,
     },
     fullName: {
-        grow: 1.3,
-        minWidth: "8rem",
+        grow: 1.2,
+        minWidth: "6.5rem",
     },
```

Comment:
The other grow values for middle columns have shrunk so full name is proportionally larger in relation to the middle columns, but smaller in relation to the last status column to account for a smaller screen